src/main/java/com/activestate/cloudfoundryjenkins/CloudFoundryPushPublisher.java

```diff
@@ -73,7 +73,7 @@ public CloudFoundryPushPublisher(String target, String organization, String clou
         this.cloudSpace = cloudSpace;
         this.credentialsId = credentialsId;
         this.selfSigned = selfSigned;
-        this.resetIfExists = resetIfExists;
+        this.cutoverMethod = cutoverMethod;
```

Comment:
The constructor is missing a `cutoverMethod` attribute, so this field won't actually be initialized. Replace `resetIfExists` on line 69.